pages/components/Card/card.html

```diff
@@ -0,0 +1,180 @@
+            <a class="brandname-link" href="/index.html">
+                <span class="brandname">
+                    <img
+                        class="brandname-logo"
+                        src="/images/logo.png"
+                        width="2em"
+                        alt=""
+                    />
+                    <span class="brandname-title">Open UI</span>
+                </span>
+            </a>
+        </header>
+        <nav class="site-nav">
+            <div class="nav-main-heading">Getting Started</div>
+
+            <div class="nav-sub-heading nav-getstarted">
+                <a href="/pages/usage.html">Usage</a>
+            </div>
+
+            <div class="nav-main-heading">Components</div>
+            <ul class="nav-comp-section">
+                <li class="component-links">
+                    <a href="/pages/components/Avatar/avatar.html">Avatar</a>
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Alert/alert.html">Alert</a>
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Badge/badge.html">Badge</a>
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Button/button.html">Button</a>
+                </li>
+                <li class="component-links">
+                    <a
+                        style="color: #027a64"
+                        href="/pages/components/Card/card.html"
+                        >Card</a
+                    >
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Image/image.html">Image</a>
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Input/input.html">Input</a>
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Lists/lists.html">List</a>
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Navigation/navigation.html"
+                        >Navigation</a
+                    >
+                </li>
+                <li class="component-links">
+                    <a href="/pages/components/Toast/toast.html">Toast</a>
+                </li>
+            </ul>
+        </nav>
+        <main class="comp-disp">
+            <h1 class="comp-disp-header">Card</h1>
+            <pre class="comp-code-container"><code class="html">
+    @import url("https://openui.netlify.app/pages/components/Card/card.css");
+    &lt;link rel="stylesheet" 
+    href="https://openui.netlify.app/pages/components/Card/card.css">
+            </code></pre>
+
+            <h2 class="sub-comp-header">Horizontal and Vertical Card</h2>
+
+            <div class="comp-disp-container disp-flex flx-wrap">
+                <div class="card">
```

Comment:
this card can have a lil bit spacing
![1616921328972](https://user-images.githubusercontent.com/61601706/112747074-ad655180-8fd0-11eb-93e5-03367b6f4e56.png)